### ISSUE TITLE
Error thrown by astro cli when used with runtime 6.0.0-alpha3

### ISSUE THREAD

**Issue by vikramkoka** [astronomer/astro-cli#707]
At the very end of:
 astro dev start

with the Dockerfile containing a reference to astro runtime 6.0.0-alpha3

after Airflow comes up, the cli reports the following error on the terminal 
-------
Airflow is starting up! This might take a few minutes…
panic: Invalid Semantic Version

goroutine 1 [running]:
github.com/Masterminds/semver/v3.MustParse(...)
	github.com/Masterminds/semver/v3@v3.1.1/version.go:214
github.com/astronomer/astro-cli/airflow.(*DockerCompose).Start(0xc000356580, {0x0, 0x0}, 0x0)
	github.com/astronomer/astro-cli/airflow/docker.go:212 +0x476
github.com/astronomer/astro-cli/cmd.airflowStart(0xc000355680?, {0x2915f40?, 0x0?, 0x0?})
	github.com/astronomer/astro-cli/cmd/airflow.go:429 +0xd0
github.com/spf13/cobra.(*Command).execute(0xc000355680, {0x2915f40, 0x0, 0x0})
	github.com/spf13/cobra@v1.2.1/command.go:856 +0x67c
github.com/spf13/cobra.(*Command).ExecuteC(0xc000354780)
	github.com/spf13/cobra@v1.2.1/command.go:974 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.2.1/command.go:902
main.main()
	github.com/astronomer/astro-cli/main.go:16 +0x31

------

By the way, the astro cli version is:
Astro CLI Version: 1.4.0

and it is running locally on my Apple Laptop running Monterey 12.5

**Comment by jedcunningham** [comment]
I believe this might be caused by logging?

```
$ docker run --rm -ti quay.io/astronomer/astro-runtime-dev:6.0.0-alpha3 airflow version
[2022-09-01 19:14:00,163] {plugin.py:79} INFO - Creating DB tables for astronomer.airflow.version_check.plugin
[2022-09-01 19:14:00,176] {plugin.py:88} INFO - Created
2.4.0.dev1554+astro.1
```

If that is the cause, this would be an issue when running 5.0.8 with DEBUG also.

**Comment by vatsrahul1001** [comment]
@jedcunningham do we have to assign this to CLI team to get this fixed

**Comment by neel-astro** [comment]
It seems like with the `6.0.0-alpha3` image we have started adding `io.astronomer.docker.airflow.version` label to the runtime image. CLI uses that label to identify the airflow version in an AC image. But it looks like the version present in the label isn't semantically correct. Not sure if that's intentionally added with some purpose as the alpha2 image doesn't seem to have that label.

```
neel@Neels-MacBook-Pro airflow-test-9 % docker inspect quay.io/astronomer/astro-runtime-dev:6.0.0-alpha3 --format='{{ index .Config.Labels "io.astronomer.docker.airflow.version"}}'
2.4.0.dev1554+astro.1
neel@Neels-MacBook-Pro airflow-test-9 % docker inspect quay.io/astronomer/astro-runtime-dev:6.0.0-alpha2 --format='{{ index .Config.Labels "io.astronomer.docker.airflow.version"}}'

neel@Neels-MacBook-Pro airflow-test-9 %
```

A short-term fix to unblock testing around `astro dev` commands on this runtime image could be to simply update the airflow version label in the Dockerfile of the local project to a semantically correct version. Updated Dockerfile in your local project might look like this:
```
FROM quay.io/astronomer/astro-runtime-dev:6.0.0-alpha3

LABEL io.astronomer.docker.airflow.version=2.4.0+astro.dev1554.1
```
**Note**: I would recommend removing that label from the local Dockerfile during `astro deploy`, as I am not sure how platforms will react to an airflow version label that does not exist.

A medium-term fix for such panic could be to update CLI code to not panic in case the airflow version is invalid, but rather throw a warning message and fallback to some default logic (explained here: https://github.com/astronomer/astro-cli/issues/657)

A long-term solution to such issues could be to start using valid semantic versioning on the airflow side because such use cases around airflow versions will only grow with time.

**Comment by ashb** [comment]
> A long-term solution to such issues could be to start using valid semantic versioning on the airflow side because such use cases around airflow versions will only grow with time.

No, Python module versions ARE NOT semver. I'm saddended to see that we as a company have to re-learn this lesson every 6 months.

**Comment by vatsrahul1001** [comment]
Verified this issues has been fixed in 6.0.0 alpha 4. Ok to be closed
